cmd/cni-metrics-helper/metrics/cni_test1.data

```diff
@@ -7,6 +7,16 @@ awscni_assigned_ip_addresses 1
 # HELP awscni_aws_api_error_count The number of times AWS API returns an error
 # TYPE awscni_aws_api_error_count counter
 awscni_aws_api_error_count{api="DeleteNetworkInterface",error="InvalidParameterValue"} 14
+# HELP awscni_ec2api_req_count The number of requests made to EC2 APIs by CNI
+# TYPE awscni_ec2api_req_count counter
+awscni_ec2api_req_count{fn="AssignPrivateIpAddresses"} 1
+awscni_ec2api_req_count{fn="AttachNetworkInterface"} 1
+awscni_ec2api_req_count{fn="CreateNetworkInterface"} 1
+awscni_ec2api_req_count{fn="DeleteNetworkInterface"} 1
```

Comment:
There should be create tags isn't it?

Reply:
It's added to be tracked. But it looks like when I took the logs that function was probably not hit until then. Do we want to explicitly check for that scenario ?

Reply:
Nvm, we are incrementing the value.